modules/extensions-spring-support/src/test/resources/META-INF/documentation-extension-descriptions.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <extension-documentation>
-    <elements name="anotherConfig">
+    <elements name="another-config">
```

Comment:
does this name match the name on the model or the name in the DSL? it should be the first